### PR TITLE
Updated Readme and Contribution files

### PR DESCRIPTION
**Pull request by SusheelThapa**

### CODE REVIEW

**Comment by SusheelThapa** [comment]
@srbmaury can you label this as hacktoberfest-accepted

**Comment by srbmaury** [comment]
To be accepted in Hacktoberfest either the repository should have a
Hacktoberfest label or The PR should have a Hacktoberfest-accepted label
(any of them will do the job) and this repository already has a
Hacktoberfest label.

Please avoid writing such unnecessary comments everywhere.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Contributing to Tic Tac Toe
+
+Thank you for your interest in contributing to Tic Tak Toe! We welcome contributions from the community and appreciate your help in making our project better.
```
Comment (srbmaury) [inline CONTRIBUTING.md:3]: replace tak with tac

File: README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# <p align="center">Tic Tak Toe</p>
```
Comment (srbmaury) [inline README.md:1]: replace tak with tac

**Comment by SusheelThapa** [comment]
@srbmaury  You have already merge this branch with the same error.
If I again open pull request with readme enhancement, it can be marked as spam

**Comment by srbmaury** [comment]
Please correct it while maintaining the folder structure.

**Pull request merged**